esp/templates/survey/review.html

```diff
@@ -12,19 +12,68 @@
 <style type="text/css">
 .nocheckmark { border: 1px solid black; }
 </style>
-<br /><br />
-<h1>Survey Responses for {{ program.niceName }} </h1>
+<a name="top"></a><h1>Survey Responses for {{ program.niceName }} </h1>
+
+{% if teacher_form %}
+<form method="POST" action="{{ request.get_full_path }}">
+<h2>Search for a teacher to retrieve their specific survey results:</h2>
```

Comment:
Can there be a way to clear the search for the teacher and return to the non-filtered results page?